rs/p2p/artifact_downloader/src/fetch_artifact/download.rs

```diff
@@ -75,7 +75,7 @@ async fn rpc_handler<Artifact: PbArtifact>(
 pub struct FetchArtifact<Artifact: PbArtifact> {
     log: ReplicaLogger,
     transport: Arc<dyn Transport>,
-    priority_fn: watch::Receiver<PriorityFn<Artifact::Id>>,
+    priority_fn: watch::Receiver<FilterFn<Artifact::Id>>,
```

Comment:
We should update all these parameter/variable names If we are going with the term `filter`.

```suggestion
    filter_fn: watch::Receiver<FilterFn<Artifact::Id>>,
```